### DISCUSSION TITLE
N-Channel mosfet recommendations for port status?

### DISCUSSION THREAD

**Discussion by icd2k3** [darthcloud/BlueRetro#694] [Q&A] [answered]
Hi All!

<img width="401" alt="image" src="https://github.com/darthcloud/BlueRetro/assets/1458959/8a0dec21-f5a4-4235-9e00-2a54274a24c3">

I see that [n-channel mosfets are recommended](https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-HW2-Internal-Install-Specification#7---port-status-led) to "avoid problem at boot". I'm new to this kind of thing, and there are so many different mosfets that are different sizes, shapes, and rated for different voltages/amps etc...

Can anyone point me in the direction what I should be searching for, or of a good n-channel mosfet to use with this?

Thanks!

EDIT: It looks like [they might not event be needed?](https://github.com/darthcloud/BlueRetro/discussions/364)

**Comment by darthcloud** [comment, marked as answer]
This is mirrored from Espressif own schematic for wrover kit RGB LED:
https://dl.espressif.com/dl/schematics/ESP-WROVER-KIT_V4_1.pdf

They use LBSS138LT1G

Try it without first, if you get trouble like the ESP not booting then try remove the LED.
